archetypes/helidon/src/main/archetype/mp/common/files/src/main/resources/application.yaml.mustache

```diff
@@ -1,3 +1,6 @@
+{{#yaml-config-entries}}
```

Comment:
It is a general best-practice to do the test and the {{.}} on the same line, since it is not so vertical-white-space verbose.

example:
```
{{#header}}{{.}}
{{/header}}
```

Reply:
This is would need to be done in all files, and ported to 3.x. If this is something we want to do, we should file an issue and address with a separate pull request.

Reply:
I have filed https://github.com/helidon-io/helidon/issues/6269 for this.